extra-dev/packages/coq-hott/coq-hott.8.10.dev/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+maintainer: "Jason Gross <jgross@mit.edu>"
+homepage: "http://homotopytypetheory.org/"
+bug-reports: "https://github.com/HoTT/HoTT/issues"
+license: "BSD 2-clause"
```

Comment:
```suggestion
license: "BSD-2-Clause"
```
We're trying to switch to [SPDX](https://spdx.org/licenses/) license identifiers.